src/soft.js/api/msgs.ts

```diff
@@ -0,0 +1,302 @@
+import { toUtf8 } from '@cosmjs/encoding';
```

Comment:
👍 like this approach in general, maybe need to rename it to be more semantic - like Soft3MessageFactory